src/scripts/main.js

```diff
@@ -1,3 +1,66 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
```

Comment:
It's a good practice to check if the element exists in the DOM before you start to use it. This could prevent potential errors if the element is not found.